client/src/components/About.js

```diff
@@ -1,7 +1,27 @@
 import React from "react";
 
 const About = () => {
-  return <div>About</div>;
+  return;
```

Comment:
because you are returning here, the code below doesn't run at all - so you'll need to remove this